hummingbot/strategy/cross_exchange_market_making/order_id_market_pair_tracker.py

```diff
@@ -0,0 +1,62 @@
+from collections import OrderedDict
+from decimal import Decimal
+
+from hummingbot.core.time_iterator import TimeIterator
+
+NaN = float("nan")
+
+
+class OrderIDMarketPairTrackingItem:
+    def __init__(self, order_id: str, exchange, market_pair):
+        self.order_id = order_id
+        self.exchange = exchange
+        self.market_pair = market_pair
+        self.expiry_timestamp = NaN
+
+
+class OrderIDMarketPairTracker(TimeIterator):
```

Comment:
The `TimeIterator` class is not prepared to have pure Python subclasses.

Reply:
✅